### ISSUE TITLE
onDragLeft is not implemented?

### ISSUE THREAD

**Issue by daniel** [ascoders/react-native-image-zoom#103]
onDragLeft doesn't seem to be implemented.

https://github.com/ascoders/react-native-image-zoom/search?q=onDragLeft&unscoped_q=onDragLeft

**Comment by ArtemKolichenkov** [comment]
Yeah, it seems that the prop doesn't do anything. Will take a look at ti
